### ISSUE TITLE
Where is kube-proxy in v3.10

### ISSUE THREAD

**Issue by esevan** [openshift/origin#20308]
[provide a description of the issue]

##### Version
oc v3.10.0-rc.0+c20e215
kubernetes v1.10.0+b81c8f8
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://172.16.20.1:443
openshift v3.10.0-rc.0+e132a20-38
kubernetes v1.10.0+b81c8f8

##### Steps To Reproduce
1. Install v3.9
2. Build openshift v3.10.rc.0
3. Upgrade v3.9 cluster to v3.10 with openshift-ansible v3.10.rc.0

##### Current Result
My services are not working.
```
$ oc get svc
default                 webserver            LoadBalancer   172.30.149.39    xxxxx    80:31398/TCP              23h
$ curl 172.30.149.39
curl: (7) Failed connect to 172.30.149.39:80; Connection timeout
```
##### Expected Result
```
$ oc get svc
default                 webserver            LoadBalancer   172.30.149.39    xxxxx    80:31398/TCP              23h
$ curl 172.30.149.39
0.5811282859649509, v0.1, Hello World!, 31282
```

##### Additional Information
Cannot find kube-proxy process
in v3.9
```
# 10256 is kube-proxy health check port
$ netstat -tnlp | grep 10256
tcp6       0      0 :::10256                :::*                    LISTEN      33898/openshift
```

in v3.10
```
$ netstat -tnlp | grep 10256
NO OUTPUT
```

I couldn't even find proxyArgument in v3.10 node-config. 

origin-node service is just executing the kubelet...
```
$ systemctl status origin-node
● origin-node.service - OpenShift Node
   Loaded: loaded (/etc/systemd/system/origin-node.service; enabled; vendor preset: disabled)
   Active: active (running) since THU 2018-07-12 07:46:39 UTC; 22h ago
     Docs: https://github.com/openshift/origin
 Main PID: 108876 (hyperkube)
    Tasks: 18
   Memory: 81.1M
   CGroup: /system.slice/origin-node.service
           └─108876 /usr/bin/hyperkube kubelet --v=2 --address=0.0.0.0 --allow-privileged=tru
```

Anything I'm missing? Please send me to the guide... where is my kube-proxy?

**Comment by esevan** [comment]
Could anyone tell me about this?

**Comment by slmingol** [comment]
I was recently looking into this as well. I can see that every node reports to have Kube-Proxy via a oc describe:
```
$ oc describe nodes ocp-master01a.mydom.com | grep Kube-Proxy
 Kube-Proxy Version:         v1.9.1+a0ce1bc657
```

**Comment by esevan** [comment]
@slmingol Thanks for comment. I will check it soon after creating latest v3.10 openshift cluster. Hope it is fixed in official v3.10 release.
